src/coordinator/permissions.js

```diff
@@ -0,0 +1,98 @@
+import { toCamelCase } from '../utils/objects';
+
+/**
+ * @typedef {Object} ContxtUserPermissions
+ * @property {number[]} applicationsExplicit Application ids the user has directly been given access to
+ * @property {number[]} applicationsImplicit Application ids the user has access to from a role or being the owner
+ * @property {string[]} roles Role ids that the user belongs to
+ * @property {number[]} stacksExplicit Stack ids the user has directly been given access to
+ * @property {number[]} stacksImplicit Stack ids the user has access to from a role or being the owner
+ * @property {string} userId
+ */
+
+/**
+ * Module that provides access to contxt user permissions
+ *
+ * @typicalname contxtSdk.coordinator.permissions
+ */
+class Permissions {
+  /**
+   * @param {Object} sdk An instance of the SDK so the module can communicate with other modules
+   * @param {Object} request An instance of the request module tied to this module's audience.
+   * @param {string} baseUrl The base URL provided by the parent module
+   */
+  constructor(sdk, request, baseUrl) {
+    this._baseUrl = baseUrl;
+    this._request = request;
+    this._sdk = sdk;
+  }
+
+  /**
+   * Gets a list of user permissions for an entire organization
```

Comment:
Could you make this a little more explicit that it is all the permission models for each user in an organization? The way it is makes it a little more ambiguous to me